app/views/shared/forms/_error_summary.html.erb

```diff
@@ -7,7 +8,7 @@
       <ul class="govuk-list govuk-error-summary__list">
         <% model.errors.each do |attribute, message| %>
           <li>
-            <%= link_to message, "##{attribute}" %>
+            <%= link_to message, "##{attribute_prefix}#{attribute}" %>
```

Comment:
this will break the links from the error message in the summary to the relevant field on existing forms, definitely affects, on the provider side, applicant details and manual address.